Rakefile

```diff
@@ -22,6 +21,7 @@ PuppetLint.configuration.log_format = "%{path}:%{linenumber}:%{check}:%{KIND}:%{
 PuppetLint.configuration.send('relative')
 PuppetLint.configuration.send('disable_80chars')
 PuppetLint.configuration.send('disable_class_inherits_from_params_class')
+PuppetLint.configuration.send('disable_arrow_alignment')
```

Comment:
I think so too, but then the check will fail. it's really annoying, cause it assumes ONE SPACE after the longest parameter. if you align to something with more spaces, then it will complain (even with aligned arrows).